ingress/ingress_updater.go

```diff
@@ -29,15 +21,12 @@ func New(frontend Frontend, proxy Proxy) controller.Updater {
 }
 
 func (u *updater) Start() error {
-	prometheus.Register(attachedFrontendGauge)
 
-	frontEnds, err := u.frontend.Attach()
+	_, err := u.frontend.Attach()
```

Comment:
Can we get rid of the int now? It's an implementation detail of the elb frontend now.